crates/renderer/src/skia/gl_surface.rs

```diff
@@ -0,0 +1,176 @@
+use gl::{types::*, *};
+use glutin::config::Config;
+use glutin::{
+    config::GlConfig,
+    context::{
+        ContextApi, ContextAttributesBuilder, GlProfile, NotCurrentGlContext,
+        PossiblyCurrentContext,
+    },
+    display::{GetGlDisplay, GlDisplay},
+    surface::{
+        GlSurface, Surface as GlutinSurface, SurfaceAttributesBuilder, SwapInterval, WindowSurface,
+    },
+};
+use std::{ffi::CString, num::NonZeroU32};
+use winit::{raw_window_handle::HasWindowHandle, window::Window};
+use winit::dpi::PhysicalSize;
+
+use freya_engine::prelude::*;
+
+pub struct SkiaSurface {
```

Comment:
I would probably rename this to something like GlSurface and then have a type alias be stored in the window state instead, same thing for the metal surface `MetalSurface`